### PR TITLE
UefiPayloadPkg/Logo/Logo.c: Fallback to builtin logo

### PR DESCRIPTION
**Pull request by mkopec**
Signed-off-by: Michał Kopeć <michal.kopec@3mdeb.com>

### CODE REVIEW

**Comment by mkopec** [comment]
Made `UefiPayloadPkg/Logo/Logo.bmp` a symlink to `MdeModulePkg/Logo/Logo.bmp` - turns out git handles symlinks correctly. @miczyg1 please let me know if this is acceptable

File: UefiPayloadPkg/UefiPayloadPkg.dsc [hunk 1]
```diff
@@ -557,7 +557,7 @@
 
   UefiCpuPkg/CpuDxe/CpuDxe.inf
   MdeModulePkg/Universal/BdsDxe/BdsDxe.inf
-  UefiPayloadPkg/Logo/Logo.inf
+  MdeModulePkg/Logo/Logo.inf
```
Comment (miczyg1) [inline UefiPayloadPkg/UefiPayloadPkg.dsc]: Isn't this wrong? It won't compile your custom Logo driver now @mkopec 
Here and below applies the same comment

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: UefiPayloadPkg/UefiPayloadPkg.dsc (hunk 1 above)
Comment (mkopec) [reply UefiPayloadPkg/UefiPayloadPkg.dsc]: This .inf only describes the bmp file, i thought that if include MdeModulePkg's version here, the bitmap from there would be used in UefiPayloadPkg. Turns out it doesn't make a difference.

Reverted to `UefiPayloadPkg/Logo/Logo.inf`.

Comment (miczyg1) [reply UefiPayloadPkg/UefiPayloadPkg.dsc]: In that case both versions should work although it would have to be tested

**Pull request merged**